test/server/wtr-config.test.js

```diff
@@ -0,0 +1,120 @@
+import { createConfig, getBrowsers, WTRConfig } from '../../src/server/wtr-config.js';
+import { createConfig as createConfigPublic, getBrowsers as getBrowsersPublic } from '../../src/index.js';
+import { expect } from 'chai';
+import sinon from 'sinon/pkg/sinon-esm.js';
```

Comment:
I'd suggest explicitly importing the things you need, and you can just use the base import now.

```suggestion
import { assert, restore, stub } from 'sinon';
```